### PR TITLE
Add articles uuid choices to ArticleRead model

### PR DESCRIPTION
**Pull request by sdonk**

### CODE REVIEW

File: exportreadiness/models.py [hunk 1]
```diff
@@ -7,6 +7,12 @@
 
 from api.model_utils import TimeStampedModel
 
+#  ('8b3f0454-5821-4635-a755-9e5a0214c594', 'ANALYSE_THE_COMPETITION')
```
Comment (konradko) [inline exportreadiness/models.py]: what's this

File: exportreadiness/models.py [hunk 2]
```diff
@@ -7,6 +7,12 @@
...
+EXREAD_ARTICLES_CHOICES = tuple(
+    [(getattr(exred_articles, item), item,) for
+     item in dir(exred_articles) if not item.startswith("__")]
```
Comment (konradko) [inline exportreadiness/models.py]: what is this sorcery

File: exportreadiness/models.py (hunk 1 above)
Comment (sdonk) [reply exportreadiness/models.py]: An example of what the code below generates

File: exportreadiness/models.py (hunk 2 above)
Comment (sdonk) [reply exportreadiness/models.py]: Yeah, not a fan of it either. Explicit is better than implicit, I suppose

**Pull request merged**